src/modules/emails/emails.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@nestjs/common';
+import { PostEmailsResponseDto } from '@ukef/helper-modules/govuk-notify/dto/post-emails-response.dto';
+import { GovukNotifyService } from '@ukef/helper-modules/govuk-notify/govuk-notify.service';
+import { PostEmailsRequestItemDto } from '@ukef/modules/emails/dto/post-emails-request.dto';
+
+@Injectable()
+export class EmailsService {
+  constructor(private readonly govukNotifyService: GovukNotifyService) {}
+
+  sendEmail(govUkNotifyKey, postEmailRequestItem: PostEmailsRequestItemDto): Promise<PostEmailsResponseDto> {
```

Comment:
`govUkNotifyKey: String`?

Reply:
`Promise<PostEmailsResponseDto | Exceptions...>`?

Reply:
Added string type and exceptions